docs/source/howto/data.rst

```diff
@@ -260,6 +260,359 @@ Finding data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+    $ verdi group show test_group
+
+    -----------------  ----------------
+    Group label        test_group
+    Group type_string  user
+    Group description  <no description>
+    -----------------  ----------------
+    # Nodes:
+    PK    Type    Created
+    ----  ------  ---------------
+     1    Code    26D:21h:45m ago
+
+Remove nodes from a Group
+^^^^^^^^^^^^^^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+    $ verdi group remove-nodes -G test_group 1
+    Do you really want to remove 1 nodes from Group<test_group>? [y/N]: y
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='test_group')
+
+    In [2]: group.remove_nodes([load_node(1)])
+
+Alternatively, you might want to remove *all* nodes from the group.
+In the command line you just need to add ``-c/--clear`` option to ``verdi group remove-nodes ..``
+
+.. code-block:: console
+
+    $ verdi group remove-nodes -c -G test_group
+    Do you really want to remove ALL the nodes from Group<test_group>? [y/N]:
+
+In the python interface you can use ``.clean()`` method to achieve the same goal:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='test_group')
+
+    In [2]: group.clear()
+
+
+Rename Group
+^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+      $ verdi group relabel test_group old_group
+      Success: Label changed to old_group
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='old_group')
+
+    In [2]: group.label = 'another_group'
+
+
+Delete Group
+^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: ipython
+
+      $ verdi group delete another_group
+      Are you sure to delete Group<another_group>? [y/N]: y
+      Success: Group<another_group> deleted.
+
+.. note::
+    Any deletion operation related to groups won't affect the nodes themselves.
+    For example if you delete a group, the nodes that belonged to the group will remain in the database.
+    The same happens if you delete nodes from the group -- they will remain in the database but won't belong to the group anymore.
+
+Copy one group into another
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This operation will copy the nodes of the source group into the destination group.
+If the destination group does not yet exist, it will be created automatically.
+
+From the command line interface:
+
+.. code-block:: console
+
+    verdi group copy source_group dest_group
+    Success: Nodes copied from group<source_group> to group<dest_group>
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: src_group = Group.objects.get(label='source_group')
+
+    In [2]: dest_group = Group(label='destination_group').store()
+
+    In [3]: dest_group.add_nodes(list(src_group.nodes))
+
+
+Organizing data with groups
+---------------------------
```

Comment:
```suggestion
Examples for using groups
-------------------------
```

Reply:
So, I think the examples in this section are useful, but they read too much like a tutorial and I'm a bit concerned that the section is overall too long already. I think we can keep them *if* we can convert them into proper How-to guides by making them overall more concise, ensuring that each sub-section shows a solution to a specific problem, and transforming the language such that the user does not get the impression that they are expected to follow along *step-by-step*.